src/get-user-code-frame.js

```diff
@@ -0,0 +1,67 @@
+// We try to load node dependencies
+let readFileSync = null
+let codeFrameColumns = null
+
+try {
+  const nodeRequire = module && module.require
+  readFileSync = nodeRequire.call(module, 'fs').readFileSync
+  codeFrameColumns = nodeRequire.call(module, '@babel/code-frame')
+    .codeFrameColumns
+} catch {
+  // We're in a browser environment
+  /* istanbul ignore next */
```

Comment:
Given that these lines run while the module is loaded, I don't know of a practical way to mock them for a single test to force them to throw. That's why I'm ignoring coverage for now. If you know of a way to do this, I'd appreciate the guidance :pray: